docs/src/main/paradox/cluster-http-management.md

```diff
@@ -207,6 +207,18 @@ Example response:
 
     Downing akka.tcp://test@10.10.10.10:111
 
+### Get /cluster/shard responses
+
+| Response code | Description
+| ------------- | -----------
+| 200           | Shard type information in JSON format
+
+Example response:
+
+{
+  "types": ["TestShardRegion"]
```

Comment:
In the Scala/Java API we call that entityTypeKey and it's typically not named "...Region".
Better examples could be "ShoppingCart", "Product".

Shall we also name that field "entityTypes" or "entityTypeKeys" instead of "types"?

Reply:
Got it - I've updated this to `entityTypeKeys` throughout.